### PR TITLE
feat: add pvc to clickhouse

### PR DESCRIPTION
**Pull request by Mokto**
Context : after a Kubernetes upgrade I lost all my tables

### CODE REVIEW

**Comment by Mokto** [comment]
Sadly the clickhouse chart doesn't allow us not to use to storageClass.

File: sentry/values.yaml [hunk 1]
```diff
@@ -234,6 +234,14 @@ clickhouse:
   enabled: true
   clickhouse:
     imageVersion: "19.16"
+    persistentVolumeClaim:
+      enabled: true
+      dataPersistentVolume: 
+        enabled: true
+        accessModes: 
+        - "ReadWriteOnce"
+        storageClassName: "standard"
```
Comment (DandyDeveloper) [inline sentry/values.yaml]: Common practice is to set this as `"default"`. May be better?

**Review by DandyDeveloper — APPROVED** [review]
Suggestion, but not essential.

File: sentry/values.yaml (hunk 1 above)
Comment (Mokto) [reply sentry/values.yaml]: It's unclear in the k8s docs, but I think it's only when you don't specify "storageClassName" that default is used. Could you confirm?

Comment (DandyDeveloper) [reply sentry/values.yaml]: Sorry completely missed this reply... Oops. Basically no "baremetal" cluster comes with a storage class, so blank typically defaults to `default` in my experience... Beyond that though really it's down to the user. I don't think it's a big deal either way.

**Comment by Mokto** [comment]
@DandyDeveloper I removed the storageClass per default, by cloning the clickhouse chart inside our repo.

**Comment by DandyDeveloper** [comment]
> @DandyDeveloper I removed the storageClass per default, by cloning the clickhouse chart inside our repo.

:+1:

**Pull request merged**